app/models/rule.rb

```diff
@@ -24,7 +26,7 @@ def from_oscap_object(oscap_rule)
     self.rationale = oscap_rule.rationale
     self.description = oscap_rule.description
     self.severity = oscap_rule.severity
-    self
+    self.rule_references = RuleReference.from_oscap_objects(oscap_rule.references)
```

Comment:
This doesn't work with AR import. I still need to figure out how to save the rule references and then associate them with the rule.